components-atomic/image/Image.js

```diff
@@ -28,17 +29,24 @@ const ImageComponent = ({
   useNextImage = false,
   imageProps = {},
 }) => {
-  if (!photo) {
-    return ""; //@TODO add default blank image
+  const { webInstance } = useWebInstance();
+  let image = photo;
+
+  // if photo does not exist, use webinstance placeholder
+  if (!image && webInstance?.placeholder) {
+    image = {...webInstance?.placeholder};
+    image.path += "/" + image?.id;
+
+    console.log(image);
```

Comment:
@rasevp 
ad1 - zabudnuty console log
ad2 - nesposobi to rerender vsade na obrazovke, ked sa vola hook?

Reply:
- console.log vymazany
- pozeral som to cez `whyDidYouRender` a nehlasilo mi to ziaden rerender obrazku

Reply:
ok